src/components/pipelines-overview/dateTime.ts

```diff
@@ -63,6 +63,44 @@ export const getXaxisValues = (timespan: number): number[] => {
   return xValues.slice(0, numDays);
 };
 
+export const getDropDownDate = (timespan: number): Date => {
+  if (!timespan) return new Date();
+  const oneDayDuration = parsePrometheusDuration('1d');
+  const numDays = Math.round(timespan / oneDayDuration);
+  const d = new Date(Date.now());
+  d.setHours(0, 0, 0, 0);
+  if (numDays != 1) {
+    d.setDate(d.getDate() - numDays);
+  }
+  return d;
+};
+
+export const formatTime = (time: string): string => {
+  if (time == undefined) {
+    return '0 seconds';
+  }
+  const t = time.split(/[:.]+/);
+  console.log('separe', t);
+  let timestring = '';
+  if (t[0] != '00') {
+    timestring = t[0].replace(/^0+/, '');
+    timestring += t[0] == '1' ? ' hour ' : ' hours ';
+  }
+  if (t[1] != '00') {
+    timestring += t[1].replace(/^0+/, '');
+    timestring += t[1] == '01' ? ' minute ' : ' minutes ';
+  }
+  if (t[2] != '00') {
+    timestring += t[2].replace(/^0+/, '');
+    timestring += t[2] == '1' ? ' second ' : ' seconds';
+  }
+  if (timestring == '') {
+    timestring = '0 seconds';
+  }
```

Comment:
better to use short forms of units like m for minute, h for hour, and s for second

Reply:
On the regular pipelines page it uses full forms

Reply:
Made the changes, review

Reply:
Here we have less space to accommodate the full forms